stripper/ze_doom3_v2.cfg

```diff
@@ -156,4 +137,560 @@ modify:
+	{
+		"OnEntitySpawned" "!selfKill1-1"
+	}
+}
+; Fix bfg relay timing exceeding item cooldown
+modify:
+{
+	match:
+	{
+		"classname" "logic_relay"
+		"targetname" "bfg_relay"
+	}
+	delete:
+	{
+		"OnTrigger" "item_bfg_launcher_modelColor255 255 255 19-1"
+	}
+	insert:
+	{
+		"OnTrigger" "item_bfg_launcher_modelColor255 255 255 15-1"
+	}
+}
+; Remake bfg button as npcclip instead of nodraw (seems like flag 16384 didnt work well)
+filter:
+{
+	"classname" "func_button"
+	"targetname" "test_button1"
+}
+add:
+{
+	"model" "*27"
+	"classname" "func_button"
+	"targetname" "test_button1"
+	"spawnflags" "1025"
+	"origin" "-7376.08 14212.5 -4028.76"
+	"vscripts" "testmike/button.nut"
+	"wait" "0"
+	"min_use_angle" "0.8"
+	"parentname" "test_center_move1"
+	"onpressed" "!selfRunScriptCodefilter();0-1"
+	"onuser4" "bfg_relayTrigger0.05-1"
+}
+; Remake rocket button as npcclip (same reasoning as above)
+filter:
+{
+	"classname" "func_button"
+	"targetname" "test_button"
+}
+add:
+{
+	"model" "*27"
+	"classname" "func_button"
+	"targetname" "test_button"
+	"spawnflags" "1025"
+	"origin" "-7203.01 13380.5 -4145.76"
+	"vscripts" "testmike/button.nut"
+	"wait" "0"
+	"min_use_angle" "0.8"
+	"parentname" "test_center_move"
+	"onpressed" "!selfRunScriptCodefilter();0-1"
+	"onuser4" "rocket_relayTrigger0.05-1"
+}
+; Fix rocket relay timing exceeding item cooldown. Also lock the button on use.
+modify:
+{
+	match:
+	{
+		"classname" "logic_relay"
+		"targetname" "rocket_relay"
+	}
+	delete:
+	{
+		"ontrigger" "item_rocket_launcher_modelSetAnimationrocketidle6.4-1"
+	}
+	insert:
+	{
+		"ontrigger" "item_rocket_launcher_modelSetAnimationrocketidle6-1"
+		"ontrigger" "test_buttonLock0-1"
+	}
+}
+; Add AFK tp to start of stage 2
+add:
+{
+	"targetname" "strp_tp_stg2_start"
+	"classname" "trigger_multiple"
+	"startdisabled" "1"
+	"wait" "0"
+	"spawnflags" "1"
+	"UseLandmarkAngles" "1"
+	"origin" "-7392 12440 -4044.5"
+	"OnStartTouch" "!activatorAddOutputorigin -6956 14144 -41240-1"
+}
+modify:
+{
+	match:
+	{
+		"classname" "trigger_once"
+		"hammerid" "3194"
+	}
+	insert:
+	{
+		"OnStartTouch" "strp_tp_stg2_startEnable101"
+	}
+}
+; Add tp avoidance stage 2 start area (1)
+add:
+{
+	"classname" "trigger_teleport"
+	"targetname" "strp_stg2_tp_avoid"
+	"spawnflags" "1"
+	"origin" "-7120 14200 -3822.5"
+	"startdisabled" "1"
+	"target" "teler1"
+	"UseLandmarkAngles" "1"
+}
+modify:
+{
+	match:
+	{
+		"classname" "func_button"
+		"hammerid" "2786"
+	}
+	insert:
+	{
+		"onpressed" "strp_stg2_tp_avoidEnable321"
+	}
+}
+; Add tp avoidance stage 2 (2)
+add:
+{
+	"model" "*364"
+	"classname" "trigger_teleport"
+	"targetname" "strp_stg2_tp_avoid_2"
+	"spawnflags" "1"
+	"origin" "-4407.07 10399.8 -5251"
+	"startdisabled" "1"
+	"target" "teler1"
+	"UseLandmarkAngles" "1"
+}
+modify:
+{
+	match:
+	{
+		"classname" "trigger_once"
+		"hammerid" "3634"
+	}
+	insert:
+	{
+		"OnStartTouch" "strp_stg2_tp_avoid_2Enable101"
+	}
+}
```

Comment:
It's usually recommended to hook into the existing TP targetname that you're filling the missing TP coverage for, so you don't have to add a new Enable input for your trigger like this. I assume you can just rename this porttele4, considering the timing with the existing tp enable input already on this trigger_once.